devops/debian/postinst

```diff
@@ -55,6 +55,8 @@ if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-decon
 		# when /usr/sbin/policy-rc.d fails, use systemctl directly
 		systemctl enable 'osconfig.service' >/dev/null || true
 		systemctl $_dh_action 'osconfig.service' >/dev/null || true
+		systemctl enable 'osconfig-platform.service' >/dev/null || true
```

Comment:
>systemctl enable 'osconfig-platform.service' >/dev/null || true [](http://example.com/codeflow?start=2&length=63)

We do not need this. The osconfig service will enable and start the osconfig-platform service when necessary. 

However, if we want to enable platform-service by default, that won't break anything.